wifiphisher.py

```diff
@@ -869,6 +869,14 @@ def get_hostapd():
         else:
             sys.exit(('[' + R + '-' + W + '] hostapd' +
                      'not found in /usr/sbin/hostapd'))
+    if not os.path.isfile('/usr/sbin/hostapd'):
+        sys.exit((
+            '\n\033[91mUnable to install hostapd from the source!\033[0m\n' +
```

Comment:
We never tried to install hostapd from the source. The error message should be something like "Unable to install the 'hostapd' package!"